tests/lib/prepare.sh

```diff
@@ -103,18 +106,9 @@ prepare_classic() {
         exit 1
     fi
 
-    mkdir -p /etc/systemd/system/snapd.service.d
```

Comment:
These changes seem to come from another PR, right? Do you need to build on top of it or can the changes in this one be reviewed separately?